### PR TITLE
add "powered by pvlib" logos

### PR DESCRIPTION
**Pull request by cwhanse**
- ~~[X] Closes #xxxx~~
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - ~~[X] Tests added~~
 - ~~[x] Updates entries to [`docs/sphinx/source/api.rst`](https://github.com/pvlib/pvlib-python/blob/master/docs/sphinx/source/api.rst) for API changes.~~
 - [X] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/master/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - ~~[x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.~~
 - [X] Pull request is nearly complete and ready for detailed review.
 - [X] Maintainer: Appropriate GitHub Labels and Milestone are assigned to the Pull Request and linked Issue.

Adds "powered by pvlib" logos to images, and adds request to use these logos to README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,8 +114,11 @@ The long-term success of pvlib-python requires substantial community support.
 License
 =======
 
-BSD 3-clause
+BSD 3-clause.
 
+If you use pvlib-python in a commercial or publicly-avaiable application, please consider displaying one of the "powered by pvlib" logos:
+<img src="docs/sphinx/source/_images/pvlib_powered_logo_horiz.png" width="300">
```
Comment (wholmgren) [inline README.md:120]:
```suggestion

<img src="docs/sphinx/source/_images/pvlib_powered_logo_horiz.png" width="300">
```

**Review by wholmgren — COMMENTED** [review]
Nice! 

Rendering available here: https://github.com/cwhanse/pvlib-python/tree/logos

Is there a good place to put them in the main documentation? Perhaps in the citing section of https://pvlib-python.readthedocs.io/en/stable/? code in [`docs/sphinx/source/index.rst`](https://github.com/pvlib/pvlib-python/blame/f318c1c1527f69d9bf9aed6167ca1f6ce9e9d764/docs/sphinx/source/index.rst#L37-L38)

File: README.md (hunk 1 above)
Comment (wholmgren) [reply README.md:120]: I think the blank line will improve layout

File: docs/sphinx/source/index.rst [hunk 2]
```diff
@@ -34,6 +33,19 @@ application.
 There is a :ref:`variable naming convention <variables_style_rules>` to
 ensure consistency throughout the library.
 
+License
+=======
+
+`BSD 3-clause <https://github.com/pvlib/pvlib-python/LICENSE>`.
```
Comment (kanderso-nrel) [inline docs/sphinx/source/index.rst:39]:
```suggestion
`BSD 3-clause <https://github.com/pvlib/pvlib-python/LICENSE>`_.
```

I think that underscore will solve this:

image

Comment (mikofski) [reply docs/sphinx/source/index.rst:39]: aha it's restructured text

**Comment by mikofski** [comment]
FYI: sine the "powered by" is in black, it doesn't show up on dark backgrounds. Should we also have a pair for light background. Lots of folks are using dark mode in GitHub, they may not see it so easily. Here's how it looks for me:
image
I have to sign out to see it:
image

**Comment by mikofski** [comment]
Also, TBH, there's not much difference between the horizontal and vertical, that is the "powered by" vertical version is not as vertical as the plain version.

**Comment by wholmgren** [comment]
How do we feel about putting the logos in the license section of the README and the documentation? I'm a little uneasy about it. I feel like the license is exactly BSD-3, no more and no less. The logo feels much more like a citation to me.

> the "powered by" is in black, it doesn't show up on dark backgrounds. Should we also have a pair for light background. Lots of folks are using dark mode in GitHub, they may not see it so easily. 

Good idea but I recommend not holding up the PR for that.

> Also, TBH, there's not much difference between the horizontal and vertical, that is the "powered by" vertical version is not as vertical as the plain version.

Not much, but enough that I can squeeze one into a slide full of logos but not the other.

**Comment by mikofski** [comment]
>How do we feel about putting the logos in the license section of the README and the documentation? I'm a little uneasy about it. I feel like the license is exactly BSD-3, no more and no less. The logo feels much more like a citation to me.

I agree,  that troubled me too. We want to encourage logos but they're not required nor part of license.

How about a new section called logos?

**Comment by cwhanse** [comment]
My thinking was that someone interested in using pvlib would look in the license section, and someone writing about using pvlib would look at the citation section. I did not mean for the text to be in addition to the license, but rather, as a comment about using pvlib. 

If the information is in the Citing section (near the bottom of the page) my hesitation is that it may not be viewed.

I'm -1 on a new section titled Logos. Open to other ideas: rename License to Using pvlib?  Put this at the end of the top section? Move Citing up?

**Comment by wholmgren** [comment]
I could be wrong, but I am guessing that the few people interested in knowing what the license actually is are also those that will actively search for it. So I would be fine demoting it to the bottom of both the documentation and the README, especially since GitHub now highlights it in the About section of the repository.

I'm -1 on renaming License. I'm fine with a new section for Logos, though I think in the long run we might need to put them on a page with usage guidelines.

**Comment by wholmgren** [comment]
@cwhanse how do you want to proceed here? @mikofski and I agree that we should maintain the purity of the license section. For a quick win that can be modified later, I suggest putting the logos in Citing, then moving License to the bottom of the page. I'm also happy to push to the PR if you're done trying to guess at what would make me happy.

**Comment by cwhanse** [comment]
@wholmgren I agree with your suggestion, just haven't made time to execute it. Feel free to push to this PR.

**Comment by mikofski** [comment]
>I suggest putting the logos in Citing, then moving License to the bottom of the page

+1

**Comment by cwhanse** [comment]
I approve. Thanks @wholmgren

**Pull request merged**